docs/codeql/ql-language-reference/formulas.rst

```diff
@@ -164,6 +164,33 @@ If the call resolves to a predicate without result, then the call is a formula.
 It is also possible to call a predicate with result. This kind of call is an
 expression in QL, instead of a formula. For more information, see ":ref:`calls-with-result`."
 
+If a call contains a dot, then the expression before the dot is called the
+receiver of the call. For instance, ``x`` is the receiver for the call ``x.isEven()``.
+For member predicate calls where the receiver is the variable ``this`` the
+receiver may be omitted. For instance, in the following example, the
```

Comment:
You introduce the concept of a receiver as part of the syntax in the first sentence. But in this sentence, you use it as a semantic thing that can be syntactically omitted. That's not really consistent.

I think logical order would be

1. receiver is something that is needed when calling member predicates.
2. in the syntax, receivers are before the dot
3. `this` receiver can be implicit